### PR TITLE
Combat tweaks

### PR DESCRIPTION
**Pull request by tommy-tran**
- Mana regen increased for every class, adjusted mana costs
- All secondary attacks cost 100% mana
- Added some range on several attacks
- Dynamite model for hunter trap
- Ranger adjusted to be able to spam (but lose a lot of damage if they do)

### CODE REVIEW

**Comment by trudeaua** [comment]
@tommy-tran Secondary attacks are 100% mana? Is the damage increased as well?

**Comment by tommy-tran** [comment]
@trudeaua Yup! 100% mana for secondary attacks but mana regen is pretty much doubled. Damage is changed a bit, will change if you guys think we should

File: Cosmic Quest - Order and Chaos/Assets/Scripts/Items/ExplosiveTrap.cs [hunk 1]
```diff
@@ -83,12 +83,18 @@ private void Detonate()
...
+            if (child.tag == "ToBeRemoved")
+            {
+                child.gameObject.SetActive(false);
+                
+                yield return new WaitForSeconds(2f);
+                child.gameObject.SetActive(true);
+                gameObject.SetActive(false);
+                break;
+            }
+        }
```
Comment (camvr) [inline Cosmic Quest - Order and Chaos/Assets/Scripts/Items/ExplosiveTrap.cs:97]: I'm curious... Why is this necessary?

Comment (tommy-tran) [reply Cosmic Quest - Order and Chaos/Assets/Scripts/Items/ExplosiveTrap.cs:97]: Just a replacement for the old mesh renderer way because the new model has a bunch of renderers

File: Cosmic Quest - Order and Chaos/ProjectSettings/TagManager.asset [hunk 2]
```diff
@@ -26,6 +26,7 @@ TagManager:
   - Ground
   - QualitySettings
   - PlayerSpawn
+  - ToBeRemoved
```
Comment (camvr) [inline Cosmic Quest - Order and Chaos/ProjectSettings/TagManager.asset:29]: Mind removing this tag since its no longer needed?
